AeroBody/Composites/knowledgecomposites.md

```diff
@@ -1,16 +1,16 @@
 Knowledge Retention
 ===
+Our team is constantly striving to improve the manufacturing process for our body shell. 
+
 ## SOPs
 Our standard operation procedures for working with composite materials in our shop are listed here:
 
-[Plug Surfacing]  
-[Fiberglass Molding]   
-[Epoxy Resin Preparation]
-[Polyester Resin Preparation]
+* [Plug Surfacing](https://github.com/Solar-Gators/docs/blob/master/AeroBody/Composites/plugprep.pdf) 
```

Comment:
I don't think you need the full Github link to the file. In the case... `[Plug Surfacing](plugprep.pdf)` should work

Reply:
Hey Raymond! when I changed the syntax to that, I just ended up getting a 404 error when testing it on livehtml. Is there anything else I might need to add?

Reply:
Hey! Can you try `[Plug Surfacing](/AeroBody/Composites/plugprep.pdf)`

Reply:
Okay! I did some testing/research and you must move the pdfs into _static to link them correctly.